clients/httpClient.go

```diff
@@ -73,6 +75,44 @@ func (hcw *httpClientWrapper) CallGetRestEndPoint(
 	return ioutil.ReadAll(resp.Body)
 }
 
+// CallPostRestEndPoint calls an external end point
+func (hcw *httpClientWrapper) CallPostRestEndPoint(
+	address string,
```

Comment:
could have been only one string param here as we do a + anyway (valid also for the get method).
Can be left as it is for now

Reply:
added a TODO for this